plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -703,18 +703,37 @@ open class HtmlRenderer(
         }
     }
 
-    override fun FlowContent.buildText(textNode: ContentText) =
+    override fun FlowContent.buildText(textNode: ContentText) = buildText(textNode, textNode.style)
+
+    private fun FlowContent.buildText(textNode: ContentText, unappliedStyles: Set<Style>) {
         when {
             textNode.extra[HtmlContent] != null -> {
                 consumer.onTagContentUnsafe { raw(textNode.text) }
             }
-            textNode.hasStyle(TextStyle.Indented) -> {
+            unappliedStyles.contains(TextStyle.Indented) -> {
                 consumer.onTagContentEntity(Entities.nbsp)
-                text(textNode.text)
+                buildText(textNode, unappliedStyles - TextStyle.Indented)
+            }
+            unappliedStyles.size == 1 && unappliedStyles.contains(TextStyle.Cover) -> buildBreakableText(textNode.text)
+            unappliedStyles.isNotEmpty() -> {
+                val styleToApply = unappliedStyles.first()
+                applyStyle(styleToApply){
+                    buildText(textNode, unappliedStyles - styleToApply)
+                }
             }
-            textNode.hasStyle(TextStyle.Cover) -> buildBreakableText(textNode.text)
             else -> text(textNode.text)
         }
+    }
+
+    private inline fun FlowContent.applyStyle(styleToApply: Style, crossinline body: FlowContent.() -> Unit){
+        when(styleToApply){
+            TextStyle.Bold -> b { body() }
+            TextStyle.Italic -> i { body() }
+            TextStyle.Strikethrough -> samp { body() }
```

Comment:
Strikethrough is `<strike>`

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/15652452/125086112-9b18aa80-e0cb-11eb-828f-807947ca2880.png)

I had to make my own tag, since i am not gonna wait for kotlinx.html's release cycle: https://github.com/Kotlin/kotlinx.html/pull/184